WickedEngine/wiScene_BindLua.cpp

```diff
@@ -701,6 +702,35 @@ int Scene_BindLua::Merge(lua_State* L)
 	return 0;
 }
 
+int Scene_BindLua::Entity_FindAllEntities(lua_State* L)
```

Comment:
The c++ name of this is just `FindAllEntities()` so it would be better if lua followed the same naming.

Reply:
I think i've done this